.github/workflows/merge-dependabot-reusable.yaml

```diff
@@ -80,6 +86,8 @@ jobs:
 
       - name: Find the release version major
         shell: bash
+        env:
+          MAVEN_ARGS: ${{ inputs.maven-args }}
```

Comment:
Why do we need to repeat the `env` block here again?

Reply:
Nice catch! We don't.

Fixed in https://github.com/apache/logging-parent/pull/266/commits/20fbfaead58687c0c7cbf9e19311ddab212c1160